index.html

```diff
@@ -54,7 +54,7 @@ <h3 class="connect-text">LET'S CONNECT</h3>
             <a href=""
```

Comment:
- [ ] [OPTIONAL] Kindly add URL links to all the available `a` tags to improve your page accessibility and reduce broken links.
![Screen Shot 2022-05-04 at 13 40 34](https://user-images.githubusercontent.com/28307960/166674711-c4237da8-eab8-4c99-af0b-3d83c9471e7a.png)